test/mock/mock-note-c-note.c

```diff
@@ -6,7 +6,9 @@ NoteDeleteResponse_Parameters noteDeleteResponse_Parameters;
 NoteNewCommand_Parameters noteNewCommand_Parameters;
 NoteNewRequest_Parameters noteNewRequest_Parameters;
 NoteRequest_Parameters noteRequest_Parameters;
+NoteRequestWithRetry_Parameters noteRequestWithRetry_Parameters;
 NoteRequestResponse_Parameters noteRequestResponse_Parameters;
+NoteRequestResponseWithRetry_Parameters noteRequestResponseWithRetry_Parameters;
```

Comment:
I would have named it like this :point_down:  to follow the convention, but it's pretty obvious and there are no name collisions.

```suggestion
NoteRequestAndResponseWithRetry_Parameters noteRequestAndResponseWithRetry_Parameters;
```

Just a heads up, feel free to ignore.

Reply:
I am an idiot.